diff --git a/node_modules/path-to-regexp/dist/index.js b/node_modules/path-to-regexp/dist/index.js
index 97f36e8..a2444b1 100644
--- a/node_modules/path-to-regexp/dist/index.js
+++ b/node_modules/path-to-regexp/dist/index.js
@@ -8,9 +8,10 @@ exports.pathToRegexp = pathToRegexp;
 exports.stringify = stringify;
 const DEFAULT_DELIMITER = "/";
 const NOOP_VALUE = (value) => value;
-const ID_START = /^[$_\p{ID_Start}]$/u;
-const ID_CONTINUE = /^[$\u200c\u200d\p{ID_Continue}]$/u;
-const ID = /^[$_\p{ID_Start}][$\u200c\u200d\p{ID_Continue}]*$/u;
+const ID_START = /^[$_a-zA-Z]$/;
+const ID_CONTINUE = /^[$_a-zA-Z0-9]$/;
+const ID = /^[$_a-zA-Z][$_a-zA-Z0-9]*$/;
+
 /**
  * Escape text for stringify to path.
  */
